.github/workflows/build-executable.yaml

```
@@ -86,72 +93,73 @@
-      - name: Download API artifact
-        uses: actions/download-artifact@v4
-        with:
-          name: asset-artifact
-          path: asset-sender-1.0-SNAPSHOT.jar
-      # push api artifact to s3
-      - name: copy API jars to S3 
+      - name: Upload dist folder to S3 bucket
         run: |
-          aws s3 cp --recursive asset-sender-1.0-SNAPSHOT.jar s3://${{ secrets.AWS_BUCKET_QA }}/${{ needs.extract_version.outputs.version }}/qa/deltaengine
+          cd dist
+          aws s3 cp --recursive . s3://${{ secrets.AWS_BUCKET_QA }}/${{ needs.extract_version.outputs.version }}/qa/
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)